pkg/operator/controller/ingress/controller_test.go

```diff
@@ -1016,6 +1016,125 @@ func TestTLSProfileSpecForSecurityProfile(t *testing.T) {
 	}
 }
 
+// TestValidateRoute ensures not specifying a required field results in error and a fully specified
+// route passes successfully
+func TestValidateRoute(t *testing.T) {
```

Comment:
I don't think underscore is needed. Can you please explain the reason ?

Reply:
It is a new unit testing pattern we adopted so that we can preserve the capitalization of the tested function in the test name, see https://redhat-internal.slack.com/archives/CBWMXQJKD/p1661531562511959

Reply:
file no longer present.